Update zh_CN translation

arisac opened the pull request.

arisac committed:
Update zh_CN translation

arisac committed:
Update zh_CN translation

fix incorrect translation

arisac committed:
fix translation

arisac committed:
improve translation for zh_CN

major

arisac committed:
improve translation for zh_CN

arisac commented:
@Gudahtt  I believe this is a merge-ready version

Gudahtt commented:
@arisac: Thanks for the contribution! I'll find somebody to review this as soon as the conflicts are resolved.

We recently added JSON sorting to our formatter, so now there are many conflicts. Would you mind rebasing this onto `develop` and running `yarn && yarn lint:fix`? That should fix everything.

arisac commented:
> @arisac: Thanks for the contribution! I'll find somebody to review this as soon as the conflicts are resolved.
> 
> We recently added JSON sorting to our formatter, so now there are many conflicts. Would you mind rebasing this onto `develop` and running `yarn && yarn lint:fix`? That should fix everything.

@Gudahtt thanks for notify.I've done rebase and solved conflicts. 
I noticed @dependabot[bot] is failing the CLA test. It should be added to CLA Test whitelist?

arisac committed:
add missing translation for zh-cn

Gudahtt commented:
Thanks again @arisac but something went wrong in your attempt to rebase. You've now duplicated ~56 commits from `develop`, and there are far more changes than there should be.

Could you check the "Allow edits from maintainers" checkbox on this PR? (see here: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork ). That would give me permission to fix things up myself.

Gudahtt commented:
@arisac Thanks for granting access! I've just rebased your initial 5 commits onto `develop`. It looks correct to me - let me know if anything seems off.

Gudahtt committed:
Remove unused messages

Three messages added on this branch were unused by MetaMask. They have
been removed.

Gudahtt commented:
I noticed that 3 messages were unused, so I removed them in https://github.com/MetaMask/metamask-extension/pull/9388/commits/c2153dce0750e473ab77840759a317149a4f01e5

Gudahtt committed:
Add missing changes

A few changes were made during the last attempt to update this branch.
They were unintentionally discarded when this branch was rebased. Those
changes have now been restored.

Gudahtt commented:
Ok that was my last update :sweat_smile: I had taken a closer look at your changes, and realized you had made a bunch of changes in that last attempt to update this branch. They were discarded when I rebased just the first 5 commits. I've just restored them all just as you had them.

arisac commented:
Thank you and sorry for the trouble, that was quite a lot work.

arisac committed:
Merge remote-tracking branch 'metamask/develop'

# Solve Conflicts:
#	app/_locales/zh_CN/messages.json

arisac commented:
I'm currently adding new translation strings to this PR. will update soon.

arisac committed:
add new strings to zh_cn translation

arisac commented:
I've updated this PR to sync with the latest develop branch. 
Still more improvements to be done to be a nicely localized in zh_cn translation, but I think this is a good start for the community to contribute on.

arisac committed:
Merge remote-tracking branch 'metamask/develop' into patch

Solve Conflicts:
#	app/_locales/zh_CN/messages.json

arisac committed:
update strings

remove unused strings
add new strings

rekmarks commented:
Hi @arisac, thank you for creating this monumental PR. Do you think you'd be willing to rebase it one more time? We recently received some more resources for translation-related work, and can finally review this properly.

很对不起我们花了这么长时间。🙇🏼‍♂️

arisac committed:
Merge remote-tracking branch 'metamask/develop'

Added new translation strings

# Fixed Conflicts:
#	app/_locales/zh_CN/messages.json

arisac commented:
@rekmarks done with new strings added.

I saw [ci/circleci: test-e2e-firefox](https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/7190/workflows/1ee72a0c-38f1-485f-b19b-7b0d9594b6a5/jobs/236732) is getting a failed result due to a timeout.

DDDDDanica commented:
@rekmarks @arisac  Is it possible to have a comparison file with translated Chinese sentence and original English one?
I can see some potential improvements but it is very difficult which is better without the context.
Thanks ~

rekmarks commented:
@DDDDDanica, I (or someone else at MetaMask) can make you something by tomorrow!

@arisac, if you created any comparison files while making this, please do let us know.

arisac commented:
@DDDDDanica @rekmarks I've uploaded the diff file to  https://github.com/arisira/metamask-extension/blob/diff/en-zh_CN.diff using `app/_locales/en/messages.json app/_locales/zh_CN/messages.json >> en-zh_CN.diff`

I removed description key from my translation file since it is not needed for translation, by removing the key, it also helps to diff the translation with the English file because it will pop up in the diff.

And also I've noticed #9772 changed the `gas` back to `gas`. To my opinion, it should be translated, and I'm using `燃料` instead of `天然气` since cn.etherscan.io is using `燃料` as well and I believe it's a good one.

arisac committed:
improve zh_CN translation

arisac committed:
Merge remote-tracking branch 'metamask/develop'

solve Conflicts:
#	app/_locales/zh_CN/messages.json

arisac commented on app/_locales/zh_CN/messages.json:
```diff
@@ -24,13 +30,19 @@
   },
   "accountSelectionRequired": {
-    "message": "您需要选择一个账户 ！"
+    "message": "需要选择一个账户 ！"
```
Thank you for the guideline. I've updated `你` with `您` in all strings on e86b5dabbe6b3816fb4370490cde598b9d3704ac

DDDDDanica replied:
💯

arisac commented on app/_locales/zh_CN/messages.json:
```diff
@@ -63,45 +75,85 @@
+    "message": "资产"
+  },
   "attemptToCancel": {
-    "message": "想要取消吗？"
```
Updated in e86b5dabbe6b3816fb4370490cde598b9d3704ac

arisac commented on app/_locales/zh_CN/messages.json:
```diff
@@ -63,45 +75,85 @@
   },
   "attemptToCancelDescription": {
-    "message": "确认提交该操作无法保证能够成功取消您的原始交易。如果取消成功，您将被收取上述交易费用。"
+    "message": "确认提交该操作无法保证能够成功取消你的原始交易。如取消成功，你将被收取上述交易费。"
```
Updated in e86b5dabbe6b3816fb4370490cde598b9d3704ac

arisac commented on app/_locales/zh_CN/messages.json:
```diff
@@ -3,13 +3,19 @@
   },
   "accessingYourCamera": {
-    "message": "正在获取相机使用权限..."
+    "message": "正在访问相机..."
```
Updated in e86b5dabbe6b3816fb4370490cde598b9d3704ac

arisac replied:
All `...` is updated with the Chinese punctuation `……` in e86b5dabbe6b3816fb4370490cde598b9d3704ac

arisac commented on app/_locales/zh_CN/messages.json:
```diff
@@ -657,52 +1014,88 @@
+    "message": "仅在主网（mainnet）上提供"
+  },
+  "onlyConnectTrust": {
+    "message": "只连接你信任的网站。"
```
Updated in e86b5dabbe6b3816fb4370490cde598b9d3704ac

arisac commented on app/_locales/zh_CN/messages.json:
```diff
@@ -657,52 +1014,88 @@
+    "message": "“$1”会关闭此标签，直接回到 $2"
+  },
+  "onlyAddTrustedNetworks": {
+    "message": "恶意的 Ethereum 以太坊网络提供商可以伪造区块链状态，并记录你的网络活动。只添加你信任的自定义网络。"
```
Updated in e86b5dabbe6b3816fb4370490cde598b9d3704ac

arisac commented on app/_locales/zh_CN/messages.json:
```diff
@@ -1009,165 +1469,488 @@
+    "message": "保证数额"
+  },
+  "swapAmountReceivedInfo": {
+    "message": "这是你将收到的最低数额。你也可能会收到更多，这取决于滑点。"
```
Updated in e86b5dabbe6b3816fb4370490cde598b9d3704ac

arisac commented on app/_locales/zh_CN/messages.json:
```diff
@@ -1009,165 +1469,488 @@
+    "message": "你还需 $1 $2 来完成这笔兑换"
+  },
+  "swapBetterQuoteAvailable": {
+    "message": "有一个更好的报价可用"
```
Updated in e86b5dabbe6b3816fb4370490cde598b9d3704ac

arisac commented on app/_locales/zh_CN/messages.json:
```diff
@@ -1009,165 +1469,488 @@
+    "message": "这是预估的用于完成你的兑换所使用的网络手续费。实际数额可能会根据网络条件而变化。"
+  },
+  "swapFailedErrorDescription": {
+    "message": "你的资金是安全的，仍然可以在你的钱包中使用。"
```
Updated in e86b5dabbe6b3816fb4370490cde598b9d3704ac

arisac commented:
rebased with the latested develop branch with 73bc553ecf99369a7dc09026fb87817bf5e2c2e1 since one string was removed from the English file: `yourSigRequested`

arisac commented:
@DDDDDanica request for review.

DDDDDanica reviewed (approved):
All looks good to me 👍

rekmarks reviewed (approved):
With @DDDDDanica's approval, I'm going to merge this.

Thank you so much for taking the time to do this @arisac, and thank you @DDDDDanica for reviewing!
